provider/ec2/internal/ec2instancetypes/.gitignore

```diff
@@ -0,0 +1 @@
+index.json
```

Comment:
Why is this here rather than at the top level? I don't suppose it matters, just was surprising to see.